### PR TITLE
Fix translations for NSDate+SAMAdditions

### PR DESCRIPTION
**Pull request by jlnr**
While using good old SSToolkit (<3), I've noticed that this localized string key looks wrong in several translations:

https://github.com/soffes/sstoolkit/blob/master/Resources/zh_Hant.lproj/SSToolkit.strings#L20

(Even without looking at the translations on the right hand side, it's a duplicate key.)

The more I looked at it, the more seemed odd in these categories :) Will push more commits if I get a chance, but feel free to discuss and merge/reject this already.

Nothing in this PR has been tested manually or automatically, and I have only looked at a few languages so far. I have verified my changes to the Chinese translations with a native speaker though.

### CODE REVIEW

File: SAMCategories/SAMCategories.bundle/zh_Hans.lproj/SAMCategories.strings [hunk 1]
```diff
@@ -14,17 +14,19 @@
...
 "less than a minute" = "少于一分钟";
 "%d minutes" = "%d 分钟";
-"about 1 hour" = "大约1小时";
-"about %d hours" = "大约%d 小时";
+"about 1 hour" = "大约 1 小时";
+"about %d hours" = "大约 %d 小时";
 "1 day" = "1 天";
 "%d days" = "%d 天";
-"about 1 month" = "大约1个月";
+"about 1 month" = "大约 1 个月";
```
Comment (jlnr) [inline SAMCategories/SAMCategories.bundle/zh_Hans.lproj/SAMCategories.strings:29]: Changed the spaces around arabic numerals to be consistent.

I wonder if this wouldn't be even better with a non-breaking space after the 1/%d. One can probably go crazy with these categories… :)

File: SAMCategories/SAMCategories.bundle/zh_Hant.lproj/SAMCategories.strings [hunk 2]
```diff
@@ -14,17 +14,19 @@
...
-"half a minute" = "半分鍾";
-"less than a minute" = "少于一分鍾";
-"%d minutes" = "%d 分鍾";
-"about 1 hour" = "大約1小時";
-"about %d hours" = "大約%d 小時";
-"1 day" = "1天";
+"1 minute" = "1 分鐘";
+"less than %d seconds" = "少於 %d 秒";
+"half a minute" = "半分鐘";
+"less than a minute" = "少於一分鐘";
```
Comment (jlnr) [inline SAMCategories/SAMCategories.bundle/zh_Hant.lproj/SAMCategories.strings:23]: I suspect that the characters originally used in this file (于, 分鍾) are Hong Kong variants, where I am following the standards for written Chinese in Taiwan. I'm not sure if either country is a bigger authority on Traditional Chinese. :( I'll try to ask friends from HK about this.

Comment (soffes) [inline SAMCategories/SAMCategories.bundle/zh_Hant.lproj/SAMCategories.strings:23]: I know Ruby on Rails refers to traditional as zh-CH and simplified as zh-TW. So the China way sounds good to me. Ready to merge?

Sam

Comment (jlnr) [inline SAMCategories/SAMCategories.bundle/zh_Hant.lproj/SAMCategories.strings:23]: It should be the other way around - CH is simplified. :)

In any case, both the Hong Kong and Taiwanese Wikipedia articles about time units use the same characters as I do, so I think my change is correct in both regions.

Ready to merge? Yep. I'm positive that this PR is a change for the better.

**Pull request merged**